### ISSUE TITLE
Unused media players?

### ISSUE THREAD

**Issue by HelgeStenstrom** [goxr3plus/XR3Player#49]
https://github.com/goxr3plus/XR3Player/blob/56cfe5cc09edf666055f5ebe8a25ff1038f85600/src/main/java/com/goxr3plus/xr3player/controllers/general/WelcomeScreen.java#L55

What's the purpose of the two media players? Are they leftovers from old solutions?

What are you using instead, and why?

**Comment by goxr3plus** [comment]
YES, I used to have some extra features like a background video and audio in Welcome screen, we might enable anytime in future so i left them there and all the code implementation :).

I think we will pass on Java 13 soon. I don't know what to do with JxBrowser and it's weird new non opensource licenses they have.

I might remove it all together and add JavaFX browser which though is very limited.

**Comment by HelgeStenstrom** [comment]
I have my own little project, and yours is a source of inspiration. I will need visualization (like an oscilloscope and/or a spectrogram) visualizing playing music or sound from a microphone. But I haven't yet figured out how to update a graph from a sound channel. That's why I looked at MediaPlayer. What package or class are you using instead of MediaPlayer? (Yes, I can find it myself by reading the source code, but it's easier to ask :) )

**Comment by goxr3plus** [comment]
It's actually a little bit complex to explain. You should start reading the code of XR3PlayerController class and see thr @Override methods os StreamPlayerListener, there is all the magic :)

**Comment by HelgeStenstrom** [comment]
That means mainly the progress() method in StreamController, which uses an instance of XPlayerController. Right?

**Comment by goxr3plus** [comment]
Yes actually you can start reverting the logic from there. The progress() is passing thr PCM data to the visualizer as you can see i give also the line to the visualizer.


It's simple as that. XR3PALAYER CONTROLLER has an instance of Visualizer which it uses. HEY

check java spectrum analyserz tutorials that i created in YouTube i explain there and i havr the code ready for you.

**Comment by HelgeStenstrom** [comment]
Do you mean this one? https://www.youtube.com/watch?v=6r9Wu_OUbPc

**Comment by goxr3plus** [comment]
This one, i habe a repository here :) https://github.com/goxr3plus/Java-Spectrum-Analyser-Tutorials
